clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/UpsertAmazonLoadBalancerAtomicOperationSpec.groovy

```diff
@@ -244,6 +244,7 @@ class UpsertAmazonLoadBalancerAtomicOperationSpec extends Specification {
 
   void "should handle VPC ELB creation"() {
       description.subnetType = "internal"
+      description.isInternal = true;
```

Comment:
could you add a test where isInternal is null and subnetType is 'internal' to confirm we have maintained backwards compatibility